src/core/annotation.js

```diff
@@ -602,6 +602,14 @@ var WidgetAnnotation = (function WidgetAnnotationClosure() {
       this.setFlags(AnnotationFlag.HIDDEN);
     }
 
+    // Support ResetForm Action Widget
+    if (data.fieldType === 'Btn' && dict.get('A') !== undefined) {
+      var actionRef = dict.map.A.num;
+      var xrefObject = dict.xref.fetchIfRef(dict.map.A);
+      data.formAction = {};
```

Comment:
Yes, I understand that. You are correct that it is not used in web/ or src/. The only place it would be used is in the examples/acroforms. Would you like me to update the code in examples/acroforms? It is my understanding that this pull request is on hold until #6172 is landed. Please let me know how you would like to proceed.